test/groovy/DockerExecuteTest.groovy

```diff
@@ -41,7 +41,7 @@ class DockerExecuteTest extends BasePiperTest {
     void init() {
         bodyExecuted = false
         docker = new DockerMock()
-        JenkinsUtils.metaClass.static.isPluginActive = {def s -> new PluginMock(s).isActive()}
+        JenkinsUtils.metaClass.static.isPluginActive = { def s -> new PluginMock(s).isActive() }
```

Comment:
Lot of whitespace changes here. Makes it impossible to provide a good review in an efficient way. IMO we should avoid this.